src/components/AboutUsSection.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import Typography from "./Typography";
+export default function AboutUsSection() {
+  return (
+    <section className="bg-off-white ">
+      <div className="flex px-40 py-20">
+        {/* image */}
+        <div className="">
+          <img
+            src="src/assets/AssetsAboutPages/Mask group.png"
+            alt="About Us"
+            className="w-[40rem]"
+          />
```

Comment:
Lalu tinggal dipanggil di sini
```suggestion
          <img
            src={image}
            alt="About Us"
            className="w-[40rem]"
          />
```